test/backend.test.ts

```diff
@@ -87,6 +93,28 @@ describe("Tests ReactNativeBackend", () => {
     });
   });
 
+  describe("onReady", () => {
+    test("calls onReady callback with true if Native SDK is initialized", () => {
+      new ReactNativeBackend({
+        dsn: EXAMPLE_DSN,
+        enableNative: true,
+        onReady: ({ nativeDidInitialize }) => {
+          expect(nativeDidInitialize).toBe(true);
+        },
+      });
+    });
+
+    test("calls onReady callback with false if Native SDK was not initialized", () => {
+      new ReactNativeBackend({
+        dsn: EXAMPLE_DSN,
+        enableNative: false,
+        onReady: ({ nativeDidInitialize }) => {
+          expect(nativeDidInitialize).toBe(true);
```

Comment:
`l`: Shouldn't this be `false`?

Reply:
Huh yes this should be false, my mistake....although why did the tests pass then hmm

Reply:
Forgot that jest does not wait for callbacks, need to use the `done` parameter for it to wait.